staging/src/k8s.io/client-go/testing/actions.go

```diff
@@ -221,11 +221,12 @@ func NewUpdateSubresourceAction(resource schema.GroupVersionResource, subresourc
 	return action
 }
 
-func NewRootDeleteAction(resource schema.GroupVersionResource, name string) DeleteActionImpl {
+func NewRootDeleteAction(resource schema.GroupVersionResource, name string, opts *metav1.DeleteOptions) DeleteActionImpl {
```

Comment:
Update `DeleteActionImpl` and `DeleteAction` to make it easier to get the options:

```go
type DeleteAction interface {
	Action
	GetName() string
	GetDeleteOptions() *metav1.DeleteOptions
}
```